content/operator/ltg/index.de.md

```diff
@@ -0,0 +1,94 @@
+
+Züge mit Reservierungspflicht sind teilweise schon viele Tage im Voraus ausgebucht. In der Hochsaison und an Feiertagen zwischen Vilnius und Klaipeda bzw. Riga sind Züge teilweise 2-3 Wochen zuvor ausgebucht.
+
+Buchungsbeginn ist bei Inlandsfahrkarten 30 Tage vor Abfahrt und bei Auslandsfahrten 60 Tage vor Abfahrt.
+
+Somit ist vor **jeder** Fahrt mit FIP Freifahrtschein ein kostenloses Ticket (0er-Fahrschein) mit/ohne Pflichtreservierung oder ein FIP 50 Ticket zu erwerben, das ggf. eine kostenlose Reservierung enthält. (siehe [Ticket- und Reservierungskauf](#ticket--und-reservierungskauf)).
+{{% /highlight %}}
+
+FIP Vergünstigungen der 1. Klasse gelten in allen Klassen. FIP Vergünstigungen der 2. Klasse gelten in der 2. und 3. Klasse.
+
+Fahrgäste der 1. Klasse erhalten in Fernverkehrszügen kostenlose Getränke und einen kleinen Imbiss.
+
+Die Züge führen Wagen der 1., 2. und 3. Klasse. Mit Ausnahme einiger Saisonzüge zwischen Vilnius und Klaipeda (mit allen Klassen) weisen die Züge entweder die Kombination 1./2. oder 2./3. Klasse auf.
+
+## Ticket- und Reservierungskauf
+
+### Online
+
+{{% booking id="ltg-website" /%}}
+
+### Vor Ort
+
+Reisezentren befinden sich nur noch in Vilnius, Kaunas und Klaipeda; ob sie 2026 noch Fahrkarten verkaufen oder reine Infoschalter werden, war Ende 2025 noch nicht absehbar.
+
+Die inzwischen an vielen Stationen aufgestellten Fahrkartenautomaten geben keine FIP-Fahrscheine aus (Stand: 11/2025).
+
+### Im Zug
+
+FIP-ermäßigte Fahrkarten können nur im Zug gekauft werden, wenn die Abfahrt von einem Bahnhof erfolgt, an dem der Fahrkartenschalter geschlossen und/oder der Automat defekt ist.
+
+## Ermäßigungen
+
+Mit regulären Fahrkarten reisen maximal 2 Kinder bis einschließlich 7 Jahre in der 2. Klasse kostenlos, müssen dann aber auf dem Schoß der Eltern sitzen. Wenn die Kinder bis 7 Jahre einen eigenen Sitz erhalten sollen, sind 50% vom Erwachsenentarif zu zahlen. Kinder bis 18 Jahre (oder als Schüler:innen noch bis 23 Jahre) erhalten 50% Ermäßigung auf den Erwachsenentarif. Personen ab 18 Jahren zahlen den vollen Erwachsenentarif, jedoch gibt es zahlreiche Ermäßigungen für Studierende, Senior:innen ab 80 Jahren, Soldat:innen, Chemotherapie-Patient:innen und Gruppen ab 3 Personen.
+
+## Tarifliche Besonderheiten
+
+### Spezialzüge und historische Fahrten
+
+Im Sommer betreibt LTG-Link freitags bis sonntags direkte Züge zwischen Vilnius und der Ostsee („seaside express"), die nur in Kretinga und Klaipeda halten. In Kretinga besteht Bus-Anschluss nach Palanga, in Klaipeda auf die Kurische Nehrung. Diese speziellen LTG-Busse müssen vorab gebucht werden. Als Fahrtziel (bzw. Startziel) gibt man „Nida(Klaipeda), „Juodkrante(Nida)" bzw. „Palanga(Kretinga)" ein.
```

Comment:
Gilt FIP auch in diesen LTG Bussen? Ich finde gar keine Informationen zu LTG Bussen. Sie sind nicht in der Routenübersicht zu finden: https://ltglink.lt/en/route-map. Laut Google Maps werden die Busse von [Kretingos r. sav.](http://www.kretinga.lt/) betrieben.

`Juodkrante` liefern in der Reiseauskunft der LTG Website keine Ergebnisse, "Nida(Klaipeda)" finde ich nur als "Klaipeda" und "Palanga(Kretinga)" finde ich nur als "Palanga".

Reply:
Ich hab ehrlich gesagt keine Ahnung. 😅

Also die Züge gibt es auf jeden Fall. Aber vielleicht findet man die Haltestellen und Verbindungen auf Grund der Jahreszeit nicht.

https://madeinvilnius.lt/de/transportiert/Z%C3%BCge/Der-Betrieb-des-Seaside-Express-Zuges-wird-ab-Juni-wieder-aufgenommen/

Es ist alles ein bisschen merkwürdig. Aber sonst würde ich es jetzt erstmal so drin lassen, ich glaube das ist jetzt auch nicht die alles entscheidende Info :D

Reply:
So wäre mein Vorschlag, bzgl. der Busse können wir ja nochmal in Klärung gehen, bevor wir behaupten, dass sie bei FIP dabei sind, aber es am Ende gar nicht sind. Dann lieber erstmal nichts zu schreiben:

```suggestion
Im Sommer betreibt LTG-Link freitags bis sonntags direkte Züge zwischen Vilnius und der Ostsee („seaside express"), die nur in Kretinga und Klaipeda halten und bei denen FIP akzeptiert wird. In Kretinga besteht Bus-Anschluss nach Palanga, in Klaipeda auf die Kurische Nehrung.
```